--- vigra-1.11.1/config/FindVIGRANUMPY_DEPENDENCIES.cmake~	2019-04-20 09:15:42.000000000 +0200
+++ vigra-1.11.1/config/FindVIGRANUMPY_DEPENDENCIES.cmake	2019-04-20 09:24:48.026845853 +0200
@@ -110,6 +110,7 @@
         # (may or may not be thread-safe)
         SET(BOOST_PYTHON_NAMES ${BOOST_PYTHON_NAMES}
                 # Linux with multiple Python versions
+                boost_python${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}
                 boost_python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}
                 # Gentoo
                 boost_python-${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}
