--- vigra-1.11.0/config/FindVIGRANUMPY_DEPENDENCIES.cmake.orig	2016-09-15 20:53:44.195804019 +0200
+++ vigra-1.11.0/config/FindVIGRANUMPY_DEPENDENCIES.cmake	2016-09-15 21:04:01.755796967 +0200
@@ -138,7 +138,7 @@
     ######################################################################
     IF(NOT DEFINED VIGRANUMPY_INSTALL_DIR OR VIGRANUMPY_INSTALL_DIR MATCHES "^$")
         execute_process(COMMAND ${PYTHON_EXECUTABLE} -c
-                         "from distutils.sysconfig import *; print(get_python_lib(1))"
+                         "from distutils.sysconfig import *; print(get_python_lib(plat_specific=True, prefix='/usr'))"
                           OUTPUT_VARIABLE PYTHON_SITE_PACKAGES OUTPUT_STRIP_TRAILING_WHITESPACE)
         FILE(TO_CMAKE_PATH ${PYTHON_SITE_PACKAGES} VIGRANUMPY_INSTALL_DIR)
     ENDIF()
