--- vigra-1.10.0/config/FindVIGRANUMPY_DEPENDENCIES.cmake~	2013-11-18 17:48:16.000000000 +0100
+++ vigra-1.10.0/config/FindVIGRANUMPY_DEPENDENCIES.cmake	2015-12-13 12:02:56.897894584 +0100
@@ -70,7 +70,7 @@
     ######################################################################
     IF(NOT DEFINED VIGRANUMPY_INSTALL_DIR OR VIGRANUMPY_INSTALL_DIR MATCHES "^$")
         execute_process(COMMAND ${PYTHON_EXECUTABLE} -c
-                         "from distutils.sysconfig import *; print get_python_lib(1)"
+                         "from distutils.sysconfig import *; print get_python_lib(plat_specific=True, prefix='/usr')"
                           OUTPUT_VARIABLE PYTHON_SITE_PACKAGES OUTPUT_STRIP_TRAILING_WHITESPACE)
         FILE(TO_CMAKE_PATH ${PYTHON_SITE_PACKAGES} VIGRANUMPY_INSTALL_DIR)
     ENDIF()
