--- vigra1.3.2/config/configure.in.orig	2005-01-27 21:06:04.000000000 +0100
+++ vigra1.3.2/config/configure.in	2005-04-03 00:07:14.930618800 +0200
@@ -38,7 +38,7 @@
     prefix=`pwd`
 fi
 
-AC_CONFIG_AUX_DIR($vigra_srcdir/config)
+AC_CONFIG_AUX_DIR(config)
 
 dnl-------------------------------------------------------------------
 dnl              compiler and platform configuration
@@ -48,9 +48,6 @@
 dnl AC_CYGWIN seems to be part of AC_CANONICAL_SYSTEM
 dnl AC_MINGW32 seems to be part of AC_CANONICAL_SYSTEM
 
-CFLAGS=-O
-CXXFLAGS=-O
-
 AC_ARG_WITH( cc, AC_HELP_STRING([--with-cc], [your C compiler (default=gcc)]), CC=$with_cc, [if test "x$CC" = "x" ; then CC=gcc ; fi])
 AC_ARG_WITH( cxx, AC_HELP_STRING([--with-cxx], [your C++ compiler (default=g++)]), CXX=$with_cxx, [if test "x$CXX" = "x" ; then CXX=g++ ; fi])
 
--- vigra1.3.2/src/impex/Makefile.in.orig	2004-11-09 11:40:31.000000000 +0100
+++ vigra1.3.2/src/impex/Makefile.in	2005-04-03 00:12:44.739480232 +0200
@@ -21,7 +21,7 @@
 all: $(TARGET)
 
 $(TARGET): $(OBJS)
-	$(LINKCXXLIB) -version-info 2:$(MAJOR_VERSION)$(MINOR_VERSION)$(MICRO_VERSION):0 -o $(TARGET) $(OBJS)
+	$(LINKCXXLIB) -version-info 2:$(MAJOR_VERSION)$(MINOR_VERSION)$(MICRO_VERSION):0 -o $(TARGET) $(OBJS) $(IMPEX_DEPLIBS)
 
 install: all
 	$(INSTALL) -d $(libdir)
